src/EVEMon/SkillPlanner/PlanWindow.cs

```diff
@@ -647,6 +650,40 @@ internal void UpdateSkillBrowser()
             skillBrowser.UpdateSkillBrowser();
         }
 
+        /// <summary>
+        /// Checks to see if the current contents of the clipboard is a valid list of skills.
+        /// </summary>
+        /// <param name="text">Clipboard contents.</param>
+        private bool CheckClipboardSkillQueue(string text)
+        {
+            string[] lines = text.Split(Environment.NewLine.ToCharArray(), StringSplitOptions.RemoveEmptyEntries);
+
+            // Nothing to evaluate
+            if (lines.Length == 0)
+                return false;
+
+            // Any lines not with a valid skill ending?
+            foreach (string line in lines)
+            {
+                if (String.IsNullOrEmpty(line) || !Regex.IsMatch(line, @"[I,V]{1,3}$"))
```

Comment:
You could use a capture group in this regex to get the roman numeral part of the line, and then if there is no match, or if RomanToInt == 0 then fail the parsing

Reply:
The check here is more to bin out if any line on the clipboard doesnt end with something other than I, II, III, IV or V before attempting to import it, but I guess it's not really ideal!

Working on an updated version now that does all that logic in the check section and present a warning to end user before import, however I still think it's beneficial to just not allow it to run if any one of the lines do not end in a Roman Numeral